packages/mobile/src/account/reducer.ts

```diff
@@ -188,7 +188,7 @@ export const reducer = (
     case Actions.SET_ACCOUNT_CREATION_TIME:
       return {
         ...state,
-        accountCreationTime: getRemoteTime(),
+        accountCreationTime: Date.now(),
```

Comment:
nothing to change now, but `now` should be passed from the action, reducers shouldn't have side effects.

Reply:
Good point. Went ahead and fixed it...no time like the present to chip away at some tech debt!